### PR TITLE
Integrate maps page

### PR DESCRIPTION
**Pull request by LayaTaal**
Add links on the Explore and publications page to the new maps page.

### CODE REVIEW

File: pages/map.js [hunk 1]
```diff
@@ -1,25 +1,50 @@
...
         <Head>
           <title>Texas Justice Initiative | Map</title>
         </Head>
-        <MapIframe src="https://map-viewer.texasjusticeinitiative.org/" />
+        <Layout fullWidth>
+          <Main>
+            <h1>{pageTitle}</h1>
+            <p>
+              This page contains map view of TJI datasets, including Custodial Deaths and Officer Involved Shootings in
+              Texas.
```
Comment (nholden) [inline pages/map.js]: What's the context that was missing from this page that we want to make more clear? I think most users could figure this out from existing context on the page. Is there something else missing here that we want to get across?

File: components/explore-the-data-page/DatasetDetails.js [hunk 2]
```diff
@@ -18,6 +19,9 @@ export default function DatasetDetails(props) {
         </ul>
       </div>
       <div className="col-right">
+        <Link href="/map">
+          <a className="btn btn--primary">View Map</a>
```
Comment (nholden) [inline components/explore-the-data-page/DatasetDetails.js]: When I saw this button in the same box as the download CSV button and the total count, I assumed the map would also respect the selected filters and dataset. What do you think of moving this out of this box?

Comment (LayaTaal) [reply components/explore-the-data-page/DatasetDetails.js]: I agree, this is bothering me as well and I wish we could use query strings here to select the map we want.

I'm not really sure where else to put the map link, though I definitely agree it's misleading where it is now. Do you have any ideas?

P.s. this is really a tricky UI to build!!

File: pages/map.js (hunk 1 above)
Comment (LayaTaal) [reply pages/map.js]: Yeah, I felt like we may need something here, but after adding a title and description area, I'm not convinced we do... Or what we should say.

File: components/explore-the-data-page/DatasetDetails.js (hunk 2 above)
Comment (nholden) [reply components/explore-the-data-page/DatasetDetails.js]: Agreed, it's definitely a tricky UI to work with. There's already a lot going on.

Maybe not the most exciting, but I think a simple link toward the top might make it seem less connected to the filtered data.

Screen Shot 2019-12-16 at 8 12 05 PM

Comment (nholden) [reply components/explore-the-data-page/DatasetDetails.js]: Unrelated, but that red text has a `mobile-only` class that doesn't seem to be defined for that element. I wonder if we should hide it on larger screens?

Comment (LayaTaal) [reply components/explore-the-data-page/DatasetDetails.js]: Yeah I noticed this...at one point that class was defined, I'm not sure what happened. Anyhow, I've reinstated it.

File: pages/map.js [hunk 3]
```diff
@@ -1,25 +1,44 @@
 import React from 'react';
 import Head from 'next/head';
 import styled from 'styled-components';
+import Layout from '../components/Layout';
 
 class Map extends React.Component {
   render() {
+    const pageTitle = 'Maps of TJI Data';
```
Comment (nholden) [inline pages/map.js]: Looks like we can dump this now.

File: components/explore-the-data-page/DatasetDetails.js (hunk 2 above)
Comment (nholden) [reply components/explore-the-data-page/DatasetDetails.js]: Awesome, looks great!

File: pages/map.js [hunk 4]
```diff
@@ -1,25 +1,44 @@
...
+    padding: 2em 4rem;
+    width: calc(100% - 300px);
+    flex-grow: 1;
+  }
+`;
+
 const MapIframe = styled.iframe`
-  height: calc(100vh - 153.375px);
-  width: 100vw;
+  height: calc(100vh - ${props => props.theme.desktopHeaderHeight});
```
Comment (nholden) [inline pages/map.js]: 😍

**Pull request merged**